### PR TITLE
(RE-5359) Update Ruby to 2.1.7 with OpenSSL 1.0.2d

### PR DESCRIPTION
**Pull request by melissa**

### CODE REVIEW

**Comment by haus** [comment]
Is that a new gem installed?

File: ruby/lib/ruby/gems/2.1.0/gems/test-unit-2.1.7.0/bin/testrb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env ruby
+require 'test/unit'
+exit Test::Unit::AutoRunner.run(true)
```
Comment (melissa) [inline ruby/lib/ruby/gems/2.1.0/gems/test-unit-2.1.7.0/bin/testrb]: @haus this is for the new version of ruby, I'm assuming, considering it's `test-unit-2.1.7.0`

**Comment by haus** [comment]
:+1:

**Comment by melissa** [comment]
screen shot 2015-09-18 at 9 00 51 am

Run on a windows 2003 i386 vmpooler image

**Comment by joshcooper** [comment]
:+1:

**Pull request merged**